config/routes.rb

```diff
@@ -1,7 +1,14 @@
 # frozen_string_literal: true
 
 Rails.application.routes.draw do
-  root to: 'development#index'
+  root to: redirect('/content')
+  get '/dev' => 'development#index'
+
+  get '/content' => 'content#index'
+  get '/content/new' => 'content#choose_format', as: :new_document
+  post '/content/create' => 'content#create', as: :create_content
```

Comment:
Similar story here - maybe we need to make a decision as to whether we call this "content" or "document"?